### DISCUSSION TITLE
zwift companion

### DISCUSSION THREAD

**Discussion by grrrrr** [Arcticons-Team/Arcticons#1121] [Q&A] [answered]
I was looking to submit a request for the [Zwift Companion](https://play.google.com/store/apps/details?id=com.zwift.android.prod&gl=US) app to be included but when I go into the Icon Request section or Arcticons it does not show up as a submit-able option.  

Looking at Arcticons it does say 4 missed apps though only shows 3 other apps.

Is there something I am missing or is it just a case of some apps are problematic?

edit: also looks like [Blink Comparison](https://f-droid.org/en/packages/org.proninyaroslav.blink_comparison/) does not show up as a submit-able app

**Comment by Donnnno** [comment, marked as answer]
This is a bug that we're aware of:
https://github.com/zixpo/candybar/issues/66
